trade-finance-manager-api/src/v1/api.js

```diff
@@ -499,7 +499,7 @@ const createEstoreFolders = async (eStoreFolderInfo) => {
     });
     return response.data;
   } catch (err) {
-    return err;
+    return false;
```

Comment:
needs some degree of error handling here it feels like - at least a log

Reply:
There is a log but not in this API. This endpoint calls reference data - which then uses estore controller. The Estore controller logs out errors

Reply:
ok makes sense